ipn/local.go

```diff
@@ -570,7 +570,7 @@ func (b *LocalBackend) parseWgStatus(s *wgengine.Status) EngineStatus {
 		tx += p.TxBytes
 	}
 	if len(ss) != 0 {
-		b.logf("v%v peers: %v", version.LONG, strings.Join(ss, " "))
+		b.logf("v%v data from peers: %v", version.LONG, strings.Join(ss, " "))
```

Comment:
I'd rather not include extra words in a line that prints so frequently. I think it's fine to leave this prefix unmodified.

The new-style peers line will only print peers we're connected to, and which line up with the "peer list" array rather than the network map array. However, an analytics script can tell the difference between the two: if there are any "peer list" lines, then it's new-style, otherwise it's old-style. So you don't need to mark anything in this line.